src/components/form/InviteeForm.js

```diff
@@ -0,0 +1,90 @@
+import React, {useState} from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Paper from '@material-ui/core/Paper';
+import PartnerDataTextField from './InviteeDataTextField';
+import InviterSelect from './InviterSelect';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+      width: '100%',
+      minWidth: 100,
+    },
+  },
+
+  button: {
+    backgroundColor: '#457b9d',
+    color: '#ffffff',
+    '&:hover': {
+      background: '#1d3557',
+    },
+  },
+}));
+
+export default function InviteeForm(props) {
+  const classes = useStyles();
+
+  const [partnerState, setPartnerState] = useState({
+    name: '',
+    email: '',
+    isGoogler: false,
+  });
+
+  // function: String => (Value => ())
+  const changePartnerState = (key) => {
```

Comment:
Didn't we agree on adding an exception handling here? :-)

Reply:
Added it. Sorry.